python/pyspark/pandas/groupby.py

```diff
@@ -2228,6 +2299,20 @@ def tail(self, n: int = 5) -> FrameLike:
         6    5
         9    8
         Name: b, dtype: int64
+
+        # Supports Groupby positional indexing Since pandas on Spark 3.4 (with pandas 1.4+)
+        >>> df = ps.DataFrame([["g", "g0"],
+        ...                   ["g", "g1"],
+        ...                   ["g", "g2"],
+        ...                   ["g", "g3"],
+        ...                   ["h", "h0"],
+        ...                   ["h", "h1"]], columns=["A", "B"])
+        >>> df.groupby("A").tail(-1) # doctest: +SKIP
```

Comment:
I record a jira SPARK-39150 as reminder to remove `# doctest: +SKIP` when we upgrade pandas to 1.4.x

Reply:
That's nice, thanks!